akvo/templates/project_directory.html

```diff
@@ -78,6 +78,13 @@
       {% blocktrans with start_index=page.start_index end_index=page.end_index paginator_count=paginator.count %}
           Viewing {{ start_index }} - {{ end_index }} of {{ paginator_count }} projects
       {% endblocktrans %}
+          | View
+          {% with "20 50 100" as page_sizes %}
```

Comment:
I see what you are saying, and it makes a lot of sense. I'll try to get this done.